test/unit/store/item-store.js

```diff
@@ -9,21 +9,16 @@ describe('ItemStore', () => {
     });
 
     it('should update an item', () => {
```

Comment:
Should this test be named 'should get an item'? 'Should update an item' sounds like you intend to make changes to an existing one.